apps/component-date/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  baseUrl: '/component-date',
+  steps: {
+    '/date-of-birth': {
+      fields: ['date-of-birth'],
```

Comment:
Looks like they do
`TypeError: (options.fields || []).reduce is not a function
    at Wizard._.each (/Users/Sulthan/Development/projects/HO/hof/node_modules/hof-form-wizard/lib/wizard.js:63:45)`
Do you want me to change it in hof?